src/gui/shell.cpp

```diff
@@ -326,6 +326,9 @@ void Shell::init()
 		return;
 	}
 
+	if (!m_shown) { // Defer displaying the shell until colors have been set
```

Comment:
```suggestion
	// Pull Request#1101: Defer displaying the shell until colors have been set
	if (!m_shown) { // Defer displaying the shell until colors have been set
```

Add a reference to the GH Issue (Pull Request), comment on separate line.